frontend/src/scenes/heatmaps/heatmapsBrowserLogic.ts

```diff
@@ -242,6 +249,27 @@ export const heatmapsBrowserLogic = kea<heatmapsBrowserLogicType>([
 
         onIframeLoad: () => {
             // TODO: Add a timeout - if we haven't received a message from the iframe in X seconds, show an error
+
+            // Attempt to access iframe content
+            // this lets us probe for errors like X-Frame-Origin issues
+            try {
+                const theIframe = props.iframeRef.current
+                const iframeContent = theIframe?.contentWindow?.document || theIframe?.contentDocument
+
+                // Check if we can access iframe content
+                if (iframeContent?.body.innerHTML) {
```

Comment:
I don't think this is what we want 🤔 Given that we will be posting messages back and forth, we don't actually want to ever be loading the iframe in such a way that we can access the raw html do we?

Reply:
I was imagining we would do this based on a timeout of not getting the "ready" state from the toolbar after N seconds

Reply:
🤔 this is just an attempt to probe that the iframe actually loaded 

I _thought_ that we'd always be able to access the raw html from JS but I've been wrong before 🙈 

Do you already have an alternative? I didn't see a signal from the iframe that i could hook into

Reply:
was trying to avoid having to wait for onLoad to timeout when the user can already see the request has failed